.travis/custom_release.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+set -x
+
+if [[ "${TRAVIS_BRANCH}" = "master" ]]; then
+    .travis/release.sh "ci-beta"
+    .travis/release.sh "qa-beta"
+    
+elif [[ "${TRAVIS_BRANCH}" = "master-stable" ]]; then
+    .travis/release.sh "ci-stable"
+    .travis/release.sh "qa-stable"
+
+# If current dev branch is deployment branch, push to build repo
```

Comment:
Shouldnt ci/qa be removed from this conditional.
It might be confusing as hell to have two branches potentially pushing to these envs.

I would consider removing the whole stanza.

Reply:
I wanted to allow specific pushes, to ci and qa. If that would be needed. But I can remove that if it would cause confusion.